api/bin/chainflip-ingress-egress-tracker/src/witnessing/state_chain.rs

```diff
@@ -84,6 +103,22 @@ enum WitnessInformation {
 		tx_out_id: TransactionId,
 		tx_ref: TransactionRef,
 	},
+	VaultDeposit {
+		#[serde(skip_serializing)]
+		tx_id: TransactionInIdForAnyChain,
+		deposit_chain_block_height: <AnyChain as Chain>::ChainBlockNumber,
+		input_asset: cf_chains::assets::any::Asset,
+		output_asset: cf_chains::assets::any::Asset,
+		amount: NumberOrHex,
+		destination_address: String,
+		ccm_deposit_metadata: Option<CcmDepositMetadata>,
+		deposit_details: Option<DepositDetails>,
+		broker_fee: Beneficiary<AccountId32>,
+		affiliate_fees: Affiliates<AccountId32>,
+		refund_params: ChannelRefundParameters,
+		dca_params: Option<DcaParameters>,
+		boost_fee: BasisPoints,
```

Comment:
think it has the `max_` prefix on some events like `DepositFinalised`, but not too important
```suggestion
		max_boost_fee: BasisPoints,
```